README.md

```diff
@@ -1,885 +1,24 @@
 # .NET Feature Management
 
+Feature management provides a way to develop and expose application functionality based on features. Many applications have special requirements when a new feature is developed such as when the feature should be enabled and under what conditions. This library provides a way to define these relationships, and also integrates into common .NET code patterns to make exposing these features possible. 
 
-[![Microsoft.FeatureManagement](https://img.shields.io/nuget/v/Microsoft.FeatureManagement?label=Microsoft.FeatureManagement)](https://www.nuget.org/packages/Microsoft.FeatureManagement)
-[![Microsoft.FeatureManagement.AspNetCore](https://img.shields.io/nuget/v/Microsoft.FeatureManagement.AspNetCore?label=Microsoft.FeatureManagement.AspNetCore)](https://www.nuget.org/packages/Microsoft.FeatureManagement.AspNetCore)
+## Get started
 
-Feature flags provide a way for .NET and ASP.NET Core applications to turn features on or off dynamically. Developers can use feature flags in simple use cases like conditional statements to more advanced scenarios like conditionally adding routes or MVC filters. Feature flags are built on top of the .NET Core configuration system. Any .NET Core configuration provider is capable of acting as the backbone for feature flags.
+[**Quickstart**](https://learn.microsoft.com/azure/azure-app-configuration/quickstart-feature-flag-dotnet): A quickstart guide is available to learn how to integrate feature flags from *Azure App Configuration* into your .NET applications.
 
-Here are some of the benefits of using this library:
+[**Feature Reference**](https://learn.microsoft.com/azure/azure-app-configuration/feature-management-dotnet-reference): This document provides a full feature rundown.
 
-* A common convention for feature management
-* Low barrier-to-entry
-  * Built on `IConfiguration`
-  * Supports JSON file feature flag setup
-* Feature Flag lifetime management
-  * Configuration values can change in real-time, feature flags can be consistent across the entire request
-* Simple to Complex Scenarios Covered
-  * Toggle on/off features through declarative configuration file
-  * Dynamically evaluate state of feature based on call to server
-* API extensions for ASP.NET Core and MVC framework
-  * Routing
-  * Filters
-  * Action Attributes
+[**API reference**](https://go.microsoft.com/fwlink/?linkid=2091700): This API reference details the API surface of this library.

```

Comment:
We have more than one library. Something like this?

This API reference details the API surface of the libraries contained within this repository.